lib/plugins/logviewer/admin.php

```diff
@@ -90,36 +90,39 @@ protected function displayLog()
             echo $this->locale_xhtml('nolog');
             return;
         }
+        
+        $logfileSize = filesize($logfile) / 1_048_576; // in MB
+        $lines = [];
+
+        if($logfileSize > 128) { // PHP default memory limit            
```

Comment:
Loading 128MB logfile into an array will still run into the memory limit because creating an array (and the rest of DokuWiki) will need memory, too. 

128 MB is also still ridiculously large. Heck even 5MB probably still is.

Reply:
Okay, I'm working on these changes.

Reply:
I used 2MB for this which I think is still large for showing in the UI (around 3k lines) and probably enough.